src/Core/Settings.h

```diff
@@ -150,6 +150,7 @@ class IColumn;
     \
     M(Bool, log_queries, 1, "Log requests and write the log to the system table.", 0) \
     M(LogQueriesType, log_queries_min_type, QueryLogElementType::QUERY_START, "Minimal type in query_log to log, possible values (from low to high): QUERY_START, QUERY_FINISH, EXCEPTION_BEFORE_START, EXCEPTION_WHILE_PROCESSING.", 0) \
+    M(Seconds, log_queries_min_time, 0, "Minimal time for the query to run, to get to the query_log.", 0) \
```

Comment:
One bit that I'm not sure of is the unit for this metric, from one point, unlikely that it will be used for anything less then a second, but from another, why not allow to set it in milliseconds?

@alexey-milovidov thoughts?

Reply:
It should be in milliseconds and named with `_ms` suffix, because there is a field `query_duration_ms` in query log.
And also let's rename it to `log_queries_min_query_duration_ms` to be more consistent (and because "time" is ambiguous).

Reply:
Ok